db/fakeTheData.js

```diff
@@ -1,33 +1,39 @@
 const faker = require('faker');
 const mongoose = require('mongoose');
 
+const env = require('../server/environment');
 const schema = require('./schema.js');
 
 const dbName = 'luke-chat';
-const dbURL = 'mongodb://localhost';
-const dbPort = 3012;
+const dbURL = `mongodb://${env.dbHost}`;
 
-mongoose.connect(`${dbURL}:${dbPort}/${dbName}`, {
+mongoose.connect(`${dbURL}:${env.dbPort}`, {
   useNewUrlParser: true,
   useUnifiedTopology: true,
+  dbName,
+  user: env.dbUser,
+  pass: env.dbPwd,
 })
   .then(() => {
     // console.log('connected to DB!');
 
     const Chat = mongoose.model('Chat', schema);
 
     const arr = [];
-    for (let i = 0; i < 250; i += 1) {
-      arr.push({
-        userName: faker.internet.userName(),
-        displayName: faker.internet.userName(),
-        displayNameColor: faker.internet.color(),
-        badge: faker.internet.avatar(),
-        badgeText: `${faker.lorem.word()} ${faker.lorem.word()}`,
-        message: faker.lorem.sentence(),
-        videoId: Math.floor(Math.random() * 10) + 1,
-        timeStamp: Math.random() * 10,
-      });
+    for (let videoId = 1; videoId < 100; videoId += 1) {
+      const numOfVideos = Math.floor(Math.random() * 100);
```

Comment:
I like this. I have a fixed number for video generator. I might change it. Thanks!